scripts/deploy-contracts-live.sh

```diff
@@ -141,36 +194,44 @@ deploy_contracts() {
 
   FLAGS="--broadcast-mode sync --fees 200000unls --gas auto --gas-adjustment 1.2 --node $nolus_net --chain-id $chain_id"
 
-  # upload Leaser code
-  local -r leaser_code_id=$(_store_code "$nolus_net" "$chain_id" "$nolus_home_dir" "$store_code_privileged_wallet_key" "$wasm_path/leaser.wasm"  "$admin_contract_address")
+  local are_valid
+  if ! are_valid=$(_keys_check "$nolus_home_dir"  "$nolus_net" "$dex_admin_wallet_key" "$store_code_privileged_wallet_key");  then
+    echo "Error: Failed to get address for key $dex_admin_wallet_key"
+    exit 1
+  fi
+
+  if [ "$are_valid" = "true" ]; then
+    # upload Leaser code
+    local -r leaser_code_id=$(_store_code "$nolus_net" "$chain_id" "$nolus_home_dir" "$store_code_privileged_wallet_key" "$wasm_path/leaser.wasm"  "$admin_contract_address")
 
-  # upload Lease code
-  local -r leaser_expected_address=$(_get_predictable_contract_address "$nolus_net" "$nolus_home_dir" "$admin_contract_address" "$leaser_code_id" "$protocol")
-  local -r lease_code_id=$(_store_code "$nolus_net" "$chain_id" "$nolus_home_dir" "$store_code_privileged_wallet_key" "$wasm_path/lease.wasm"  "$leaser_expected_address")
+    # upload Lease code
+    local -r leaser_expected_address=$(_get_predictable_contract_address "$nolus_net" "$nolus_home_dir" "$admin_contract_address" "$leaser_code_id" "$protocol")
+    local -r lease_code_id=$(_store_code "$nolus_net" "$chain_id" "$nolus_home_dir" "$store_code_privileged_wallet_key" "$wasm_path/lease.wasm"  "$leaser_expected_address")
```

Comment:
[here ahe below] please fix the indentation

Reply:
The only problem I see with indentation is on line 79, but not here